ocs_ci/ocs/cluster.py

```diff
@@ -383,3 +384,25 @@ def remove_mon_from_cluster(self):
         )
         logging.info(f"Removed the mon {random_mon} from the cluster")
         return remove_mon
+
+    @retry(exceptions.UnexpectedBehaviour, tries=10, delay=3, backoff=1)
```

Comment:
you'll need to rebase as this function was already moved to cluster.py with a different PR

Reply:
Done